src/app/static/src/app/components/genericChart/utils.ts

```diff
@@ -32,17 +32,17 @@ export function numeralJsToD3format(numeralJsFormat: string) {
     // We currently support only numeric, large number, and percentage formats, and will return 
     // empty string for any other formats received, for default formatting in Plotly.
 
-    if (numeralJsFormat === "0,0") {
-        return ","
-    }
-
-    const regex = /^0(\.0+)?(%)?$/; //This will always return three matches
+    const regex = /^0(\.0+)?(%)?$|^0(,0+)/; //This will always return four matches
```

Comment:
I think there should probably be a $ at the end of the second part of the regex, if only to make it consistent, and as I say it would be worth putting in an explanatory comment (as regexes are horrible to read).
And I'm not sure we want the + after ,0 do we? That implies we could have 0,00 which I don't think is valid numeralJS.  
```suggestion
     // The first part of this regex (before the |) captures formats with decimal precision and optional percentage
     // The second part captures format with thousands separator for large integers
    const regex = /^0(\.0+)?(%)?$|^0(,0)$/; //This will always return four matches
```